returnn/sprint/cache.py

```diff
@@ -313,11 +313,10 @@ def _raw_read(self, size, typ):
 
     elif typ in ["align", "align_raw"]:
       type_len = self.read_U32()
-      typ = self.read_str(type_len)
-      assert typ == "flow-alignment"
+      assert self.read_str(type_len) == "flow-alignment"
```

Comment:
Can you keep that as before (and maybe just rename the var)? That makes the error reporting much better (it will print what you got here from `read_str`).
I.e.
```suggestion
      type_ = self.read_str(type_len)
      assert type_ == "flow-alignment"
```